### ISSUE TITLE
Add option to show time slots in list view

### ISSUE THREAD

**Issue by chah90** [nielsfaber/scheduler-card#863]
### Checklist

- [X] I checked for similar existing requests (both open and closed) before posting.
- [X] My request is generic, other users may benefit from it too.
- [X] I will participate in further discussion about this feature and can test it (if requested) once it's done.

### Proposal

First of all, thanks for this great addon! I'm using it (besides other use cases) together with Versatile Thermostat to control the presets of my thermostats. I really like the time slot editor and the overview it provides. I would love to see an option to get a read only version of that view in the list view, to have all relevant information of the specific schedules at a glance.

### Additional info

I tried to visualize my idea with my limited photoshop skills:

Now:
<img width="401" alt="Now" src="https://github.com/user-attachments/assets/876f0516-5d02-4b4f-8f85-4f36731d1bde" />

Proposal:
<img width="401" alt="proposal" src="https://github.com/user-attachments/assets/0a44da1e-5e03-44c5-aa16-ea77fd4e7696" />

**Comment by garyjlo** [comment]
Yes good request, I think this is the same: #767 
Unfortunately we let that issue go stale and closed.

**Comment by Co0p3r21** [comment]
This would be awesome, for me mainly because it would allow showing the schedule together with the entity itself at one glance on a dashboard.

**Comment by garyjlo** [comment]
bump

**Comment by Co0p3r21** [comment]
Bump

**Comment by nielsfaber** [comment]
I agree the proposal could be a nice improvement but I do not have the time to implement this.
There is no point for bumping this request.
If someone is willing and has the skills to contribute to the project by making a PR for this change, that would be very helpful.
